src/Stratis.Bitcoin.Features.Wallet/WalletTransactionHandler.cs

```diff
@@ -38,6 +41,8 @@ public class WalletTransactionHandler : IWalletTransactionHandler
 
         private readonly ILogger logger;
 
+        private MemoryCache privateKeyCache = new MemoryCache(new MemoryCacheOptions(){ExpirationScanFrequency = new TimeSpan(0,1,0)});
```

Comment:
Please initialize this in the constructor, it's easier when debugging :-)

Reply:
Done